modules/ROOT/pages/propagate-metadata-studio.adoc

```diff
@@ -0,0 +1,161 @@
+= Propagate Metadata to Other Flows
+
+Studio enables you to propagate your flow's metadata to your referenced flows within your application.
+
+Depending on the metadata configured in your referenced flow, you might need to configure the specific metadata to propagate.
+
+== Propagate Metadata to a Flow Without Custom Metadata
+
+When you add a Flow Reference component to reference another flow with no custom metadata configured, Studio automatically propagates the metadata in your flow as custom metadata in your target flow.
```

Comment:
Should we put  `When you add a Flow Reference component and reference another flow` ....?